exercises/acronym/spec/acronym_spec.cr

```diff
@@ -2,31 +2,31 @@ require "spec"
 require "../src/*"
 
 describe "Acronym" do
-  it "tests basic" do
+  it "does basic" do
     Acronym.abbreviate("Portable Network Graphics").should eq("PNG")
   end
 
-  it "tests lowercase words" do
+  pending "does lowercase words" do
```

Comment:
Ah good catch here too to update the pending stuff. I'm a bit annoyed that my current make task mutates the files and doesn't change them back. Oh well. That's fixed in my PR.